ebos/eclproblem.hh

```diff
@@ -2440,7 +2439,7 @@ private:
         const auto& eclState = simulator.vanguard().eclState();
         const auto& timeMap = schedule.getTimeMap();
         const auto& initconfig = eclState.getInitConfig();
-        int episodeIdx = initconfig.getRestartStep() - 1;
+        int episodeIdx = initconfig.getRestartStep();
 
```

Comment:
I had to remove "-1" to get the right starting point for the restart. Why was the "-1" needed in the current code?

Reply:
maybe it isn't: opm-common is quite inconsistent when it uses 0-based C-style indices and 1-based FORTRAN-style ones.